update.sh

```diff
@@ -2,14 +2,19 @@
 
 set -e
 
-if [[ ! $# -eq 2 ]]; then
-	echo "usage: $0 [ruby|mruby|jruby|rubinius|truffleruby|truffleruby-graalvm] [VERSION]"
+if [[ $# -lt 2 ]]; then
+	echo "usage: $0 [ruby|mruby|jruby|rubinius|truffleruby|truffleruby-graalvm] [VERSION] [RELEASE_DIRECTORY]"
 	exit 1
 fi
 
 ruby="$1"
 version="$2"
-dest="pkg"
+dest="$3"
+if [ -z "$dest" ]; then
+	dest="pkg"
+fi
```

Comment:
This can be simplified to `dest=${3:-pkg}`.

Reply:
Done